indico/queries/submission.py

```diff
@@ -13,14 +13,15 @@
 
 class ListSubmissions(GraphQLRequest):
     """
-    List all Submissions visible to the authenticated user
+    List all Submissions visible to the authenticated user by most recent. 
 
     Args:
         submission_ids (List[int], optional): Submission ids to filter by
         workflow_ids (List[int], optional): Workflow ids to filter by
         filters (SubmissionFilter or Dict, optional): Submission attributes to filter by
         limit (int, optional): Maximum number of Submissions to return. Defaults to 1000
-
+        orderBy (str, optional): Order by field. Defaults to id
```

Comment:
just to be pythonic - shouldn't this kwarg be snake case? the variable name passed to graphql can be camel case
see how submission_ids and workflow_ids are formatted